fire/test_testutils.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test the test utilities for Fire's tests."""
+
+from __future__ import print_function
+
+import sys
+
+import six
+
+from fire import testutils
+
+
+class TestTestUtils(testutils.BaseTestCase):
+  """Let's get meta."""
+
+  def test_no_check_on_exception(self):
```

Comment:
We've been using testNames instead of test_names so far. Let's keep that for consistency.